test-case/check-kmod-load-unload.sh

```diff
@@ -50,6 +50,17 @@ do
     ## - 1: remove module section
     func_lib_setup_kernel_last_line
 
+    # 1. check dsp status before removal, only remove module when dsp is suspended.
+    # 2. it will take some time (about 10s) for dsp to become suspended from active
+    # after module insertion on some platforms, here we check dsp status, and only continue
+    # next removal when dsp is suspended.
+    dlogi "wait dsp power status to become suspended"
+    for i in $(seq 1 15)
+    do
+        [[ $(sof-dump-status.py --dsp_status) == "suspended" ]] && break
+        sleep 1
+    done
+
```

Comment:
so here you couldn't know it is out of define data or it just meet the target  
if out of data means it is not work as our expect, it is the bug?

Reply:
@Bin-QA I understand your concern. as we give a very big number to wait, here 15s, it should be big enough for dsp to be suspended(if it is in normal state, no previous failure). that means, the "out of data" condition never happens.

we can check the exit condition, too, if out of data , tag failed and exit test, but seems it never happens.

Reply:
check added, if dsp is always active, the script will failed. please review again @xiulipan @Bin-QA